README.md

```diff
@@ -69,7 +69,7 @@ Translations
 ------------
 
 Changes to translations as well as new translations can be submitted to
-[Bitcoin Core's Transifex page](https://www.transifex.com/bitcoin/bitcoin/).
+[Bitcoin Core's Transifex page](https://explore.transifex.com/bitcoin/bitcoin/).
```

Comment:
The current link works fine for me. It redirects to https://app.transifex.com/bitcoin/bitcoin/dashboard/ or https://explore.transifex.com/bitcoin/bitcoin/ depending on website cookies content.

Reply:
It works for me as well, so not absolutely needed to update it.

Reply:
It works here as well.

Reply:
If a translator has already logged into their Transifex account, the current link brings them directly to the project dashboard. But the suggested link brings to the project in "explore" mode, and logging in is required to start working on translations.

So I'm NACK about this change.